src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -584,4 +586,31 @@ private void disposeAllServers() {
         }
     }
 
+    /**
+     * Returns the modification trackers for all language server definitions for the file.
+     *
+     * @param file the PSI file
+     * @return all relevant modification trackers
+     */
+    @NotNull
+    @ApiStatus.Internal
+    public Set<ModificationTracker> getModificationTrackers(@NotNull PsiFile file) {

```

Comment:
Gets all modification trackers for the provided file so that they can be used as cache dependencies.